pyfar/samplings/__init__.py

```diff
@@ -1,7 +1,12 @@
 """
 Collection of sampling schemes and related functionality. For information on
 the used coordinate systems refer to the
-:py:mod:`concepts <pyfar._concepts.coordinates>`.
+:py:mod:`Coordinates documentation <pyfar.classes.coordinates>`.
+
+.. warning::
+
+    This module will be deprecated in pyfar v0.8.0 in favor of
+    :py:mod:`spharpy.samplings <spharpy.samplings>`.
```

Comment:
Is this intended to be part of this PR?

Reply:
It might be a beer. When checking the documentation I thought something like this would be nice to have